Source/Scene/TileImagery.js

```diff
@@ -43,13 +43,14 @@ define([
      *
      * @param {Tile} tile The tile to which this instance belongs.
      * @param {FrameState} frameState The frameState.
+     * @param {Boolean} skipLoading True to skip loading, but synchronously process imagery that's already mostly ready to go.
      * @returns {Boolean} True if this instance is done loading; otherwise, false.
      */
-    TileImagery.prototype.processStateMachine = function(tile, frameState) {
+    TileImagery.prototype.processStateMachine = function(tile, frameState, skipLoading) {
```

Comment:
Does `skipLoading` really mean `skipLODs` ? Or is it something else? If it's the former, should it be renamed?

Reply:
No, nothing to do with skipping LODs. I'll change the doc to hopefully make it clearer.